packages/opera.rb

```diff
@@ -3,7 +3,7 @@
 class Opera < Package
   description "Opera isn't just a browser. It's an awesome browser."
   homepage 'https://www.opera.com/'
-  version '72.0.3815.400'
+  version `curl -Ls https://get.geo.opera.com/pub/opera/desktop/ | grep -r 'a' index.html | tail -1 | cut -d '"' -f 2 | sed -e 's|\/||' | perl -e 'while(<>) { $_ =~ s/[\r\n]//g; print "$_" }'`
```

Comment:
Is this idea here to make sure we always get the latest version?  I'm not sure I think this is a good idea.

Reply:
I will test this package to see if it works when every new version of Opera release:)

Reply:
The more I think about this, the more I think it's a bad idea.  The idea is good but it probably would be better suited for a revision update script.  Thoughts?

Reply:
How about asking for the user at install (two options, the tested stable and latest stable)?